documentation/docs/libraries/overview.md

```diff
@@ -1,5 +1,12 @@
 ---
-slug: /libraries/overview
+description: There are Rust, Node.js, Python and Java bindings for the iota.rs clien library.
```

Comment:
```suggestion
description: There are Rust, Node.js, Python and Java bindings for the iota.rs client library.
```

Reply:
But Rust isn't a binding

Reply:
How about 
---suggestion
description: There are Node.js, Python and Java bindings for the iota.rs client Rust library.
---